### PR TITLE
Update swagger to openapi

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [![Build Status](https://travis-ci.org/ballerina-guides/open-api-based-service.svg?branch=master)](https://travis-ci.org/ballerina-guides/open-api-based-service)
 
-# Swagger / OpenAPI
+# OpenAPI
 [OpenAPI Specification](https://swagger.io/specification/) (formerly called the Swagger Specification) is a specification that creates RESTful contract for APIs, detailing all of its resources and operations in a human and machine-readable format for easy development, discovery, and integration.
-The Swagger to Ballerina Code Generator can take existing Swagger definition files and generate Ballerina services from them.
+The OpenApi to Ballerina Code Generator can take existing OpenApi definition files and generate Ballerina services from them.
```
Comment (pubudu91) [inline README.md]:
```suggestion
The OpenAPI to Ballerina Code Generator can take existing OpenAPI definition files and generate Ballerina services from them.
```

File: README.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ You'll build an RESTful web service using an OpenAPI / Swagger specification. Th
 
 > If you want to skip the basics, you can download the git repo and directly move to the "Testing" section by skipping  "Implementation" section.
 
-### Understand the Swagger / OpenAPI specification
-The scenario that we use throughout this guide will base on a [petstore.json](guide/petstore.json) swagger specification. The OpenAPI / Swagger specification contains all the required details about the pet store RESTful API. Additionally, You can use the Swagger view in Ballerina Composer to create and edit OpenAPI / Swagger files. 
+### Understanding the OpenAPI specification
+The scenario that we use throughout this guide will base on [petstore.json](guide/petstore.json) OpenApi specification. The OpenAPI specification contains all the required details about the pet store RESTful API. Additionally, You can use the Api Designer view in Ballerina vscode extension to create and edit OpenAPI files.
```
Comment (pubudu91) [inline README.md]:
```suggestion
The scenario that we use throughout this guide will be based on the [petstore.json](guide/petstore.json) OpenAPI specification. The OpenAPI specification contains all the required details about the pet store RESTful API. Additionally, You can use the API Designer view in Ballerina VSCode extension to create and edit OpenAPI files. 
```

File: README.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ The scenario that we use throughout this guide will base on a [petstore.json](gu
 }
 ```
 
-NOTE :  The above OpenAPI / Swagger definition is only the basic structure. You can find the complete OpenAPI / Swagger definition in [petstore.json](guide/petstore.json) file.
+NOTE :  The above OpenAPI definition is only the basic structure. You can find the complete OpenAPI definition in [petstore.json](guide/petstore.json) file.
```
Comment (pubudu91) [inline README.md]:
```suggestion
NOTE :  The above OpenAPI definition is only the basic structure. You can find the complete OpenAPI definition in the [petstore.json](guide/petstore.json) file.
```

File: README.md [hunk 4]
```diff
@@ -93,17 +93,17 @@ open-api-based-service
    $ ballerina init
 ```
 
-### Genarate the web service from the Swagger / OpenAPI definition
+### Genarate the web service from the OpenAPI definition
 
-Ballerina language is capable of understanding the Swagger / OpenAPI specifications. We can easily generate the web service just by typing the following command structure in the terminal.
+Ballerina language is capable of understanding the OpenAPI specifications. We can easily generate the web service just by typing the following command structure in the terminal.
```
Comment (pubudu91) [inline README.md]: This sounds like it's repeating info from a previous section. @praneesha thoughts?

Comment (pubudu91) [inline README.md]:
```suggestion
Ballerina language is capable of understanding OpenAPI specifications. We can easily generate the web service just by typing the following command structure in the terminal.
```

File: README.md [hunk 5]
```diff
@@ -248,7 +248,7 @@ Next we need to implement the business logic in the `ballerina_petstore_impl.bal
 
 ### Implement the business logic for petstore 
 
-Now we have the Ballerina web service for the give `petstore.json` Swagger file. Next task is to implement the business logic for functionality of each resource. The Ballerina Swagger generator has generated `ballerina_petstore_impl.bal` file inside the `open-api-based-service/guide/petstore`. We need to fill the `ballerina_petstore_impl.bal` as per the requirement. For simplicity, we will use an in-memory map to store the pet data. The following code is the completed pet store web service implementation. 
+Now we have the Ballerina web service for the give `petstore.json` OpenApi file. Next task is to implement the business logic for functionality of each resource. The Ballerina OpenAPi generator has generated `ballerina_petstore_impl.bal` file inside the `open-api-based-service/guide/petstore`. We need to fill the `ballerina_petstore_impl.bal` as per the requirement. For simplicity, we will use an in-memory map to store the pet data. The following code is the completed pet store web service implementation.
```
Comment (pubudu91) [inline README.md]:
```suggestion
Now we have the Ballerina web service for the given `petstore.json` OpenAPI file. Next task is to implement the business logic for functionality of each resource. The Ballerina OpenAPI generator has generated `ballerina_petstore_impl.bal` file inside the `open-api-based-service/guide/petstore` directory. We need to fill the `ballerina_petstore_impl.bal` as per the requirement. For simplicity, we will use an in-memory map to store the pet data. The following code is the completed pet store web service implementation. 
```

File: README.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
 [![Build Status](https://travis-ci.org/ballerina-guides/open-api-based-service.svg?branch=master)](https://travis-ci.org/ballerina-guides/open-api-based-service)
 
-# Swagger / OpenAPI
+# OpenAPI
 [OpenAPI Specification](https://swagger.io/specification/) (formerly called the Swagger Specification) is a specification that creates RESTful contract for APIs, detailing all of its resources and operations in a human and machine-readable format for easy development, discovery, and integration.
```
Comment (praneesha) [inline README.md]:
```suggestion
The [OpenAPI specification](https://swagger.io/specification/) (formerly called the Swagger specification) creates RESTful contracts for APIs. It details all resources and operations of APIs in a human and machine-readable format for easy development, discovery, and integration.
```

File: README.md [hunk 7]
```diff
@@ -1,10 +1,10 @@
 [![Build Status](https://travis-ci.org/ballerina-guides/open-api-based-service.svg?branch=master)](https://travis-ci.org/ballerina-guides/open-api-based-service)
 
-# Swagger / OpenAPI
+# OpenAPI
 [OpenAPI Specification](https://swagger.io/specification/) (formerly called the Swagger Specification) is a specification that creates RESTful contract for APIs, detailing all of its resources and operations in a human and machine-readable format for easy development, discovery, and integration.
-The Swagger to Ballerina Code Generator can take existing Swagger definition files and generate Ballerina services from them.
+The OpenAPI to Ballerina Code Generator can take existing OpenAPI definition files and generate Ballerina services from them.
```
Comment (praneesha) [inline README.md]:
```suggestion
The OpenAPI to Ballerina Code Generator can generate Ballerina services from existing OpenAPI definition files.

```

File: README.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ The following are the sections available in this guide.
 - [Observability](#observability)
 
 ## What you'll build
-You'll build an RESTful web service using an OpenAPI / Swagger specification. This guide will walk you through building a pet store server from an OpenAPI / Swagger definition. The pet store service will have RESTful POST,PUT,GET and DELETE methods to handle pet data.
+You'll build a RESTful web service using an OpenAPI specification. This guide will walk you through building a pet store server from an OpenAPI definition. The pet store service will have RESTful POST,PUT,GET and DELETE methods to handle pet data.
```
Comment (praneesha) [inline README.md]:
```suggestion
You will build a RESTful web service using the OpenAPI specification. This guide will walk you through building a pet store server from an OpenAPI definition. The pet store service will have the RESTful POST, PUT, GET and DELETE methods to handle pet data.
```

File: README.md [hunk 9]
```diff
@@ -37,10 +37,10 @@ You'll build an RESTful web service using an OpenAPI / Swagger specification. Th
 
 > If you want to skip the basics, you can download the git repo and directly move to the "Testing" section by skipping  "Implementation" section.
 
-### Understand the Swagger / OpenAPI specification
-The scenario that we use throughout this guide will base on a [petstore.json](guide/petstore.json) swagger specification. The OpenAPI / Swagger specification contains all the required details about the pet store RESTful API. Additionally, You can use the Swagger view in Ballerina Composer to create and edit OpenAPI / Swagger files. 
+### Understanding the OpenAPI specification
+The scenario that we use throughout this guide will be based on the [petstore.json](guide/petstore.json) OpenAPI specification. The OpenAPI specification contains all the required details about the pet store RESTful API. Additionally, You can use the API Designer view in Ballerina VSCode extension to create and edit OpenAPI files.
```
Comment (praneesha) [inline README.md]:
```suggestion
The scenario used throughout this guide is based on the [petstore.json](guide/petstore.json) OpenAPI specification. The OpenAPI specification contains all the required details about the RESTful API of the pet store. Additionally, you can use the API Designer view in the Ballerina VSCode extension to create and edit the OpenAPI files. 
```

File: README.md [hunk 10]
```diff
@@ -37,10 +37,10 @@ You'll build an RESTful web service using an OpenAPI / Swagger specification. Th
 
 > If you want to skip the basics, you can download the git repo and directly move to the "Testing" section by skipping  "Implementation" section.
 
-### Understand the Swagger / OpenAPI specification
-The scenario that we use throughout this guide will base on a [petstore.json](guide/petstore.json) swagger specification. The OpenAPI / Swagger specification contains all the required details about the pet store RESTful API. Additionally, You can use the Swagger view in Ballerina Composer to create and edit OpenAPI / Swagger files. 
+### Understanding the OpenAPI specification
+The scenario that we use throughout this guide will be based on the [petstore.json](guide/petstore.json) OpenAPI specification. The OpenAPI specification contains all the required details about the pet store RESTful API. Additionally, You can use the API Designer view in Ballerina VSCode extension to create and edit OpenAPI files. 
 
-#### Basic structure of petstore Swagger/OpenAPI specification
+#### Basic structure of petstore OpenAPI specification
```
Comment (praneesha) [inline README.md]:
```suggestion
#### Basic structure of the pet store OpenAPI specification
```

File: README.md [hunk 11]
```diff
@@ -93,17 +93,17 @@ open-api-based-service
    $ ballerina init
 ```
 
-### Genarate the web service from the Swagger / OpenAPI definition
+### Genarate the web service from the OpenAPI definition
 
-Ballerina language is capable of understanding the Swagger / OpenAPI specifications. We can easily generate the web service just by typing the following command structure in the terminal.
+Ballerina language is capable of understanding OpenAPI specifications. We can easily generate the web service just by typing the following command in the terminal.
```
Comment (praneesha) [inline README.md]:
```suggestion
Ballerina language is capable of understanding OpenAPI specifications. Execute the below command in a Terminal, to generate the web service.
```

File: README.md [hunk 12]
```diff
@@ -93,17 +93,17 @@ open-api-based-service
...
 
-Ballerina language is capable of understanding the Swagger / OpenAPI specifications. We can easily generate the web service just by typing the following command structure in the terminal.
+Ballerina language is capable of understanding OpenAPI specifications. We can easily generate the web service just by typing the following command in the terminal.
 ```
-ballerina swagger mock <swaggerFile> [-o <output directory name>] [-m <module name>] 
+ballerina openaapi gen-service <openApiFile> [-o <output directory name>] [-m <module name>] 
 ```
 
-For our pet store service we need to run the following command from the `/guide` in sample root directory(location where you have the petstore.json file) to generate the Ballerina service from the OpenAPI / Swagger definition
+For our pet store service we need to run the following command from the `/guide` in sample root directory(location where you have the petstore.json file) to generate the Ballerina service from the OpenAPI definition
```
Comment (praneesha) [inline README.md]:
```suggestion
Navigate to the `/guide` directory in the sample root directory (i.e., the location where you have the `petstore.json` file) and execute the below command. This generates the Ballerina service for the pet store service from the OpenAPI definition.
```

File: README.md [hunk 13]
```diff
@@ -93,17 +93,17 @@ open-api-based-service
...
 
 ```bash 
-$ ballerina swagger mock petstore.json -m petstore
+$ ballerina openapi gen-service petstore.json -m petstore
 ```
 
 The `-m` flag indicates the module name and `-o` flag indicates the file destination for the web service. These parameters are optional and can be used to have a customized module name and file location for the project.
```
Comment (praneesha) [inline README.md]:
```suggestion
The `-m` flag denotes the module name and the `-o` flag indicates the destination of the file corresponding to the web service. These parameters are optional and can be used to have a customized module name and file location for the project.
```

File: README.md [hunk 14]
```diff
@@ -248,7 +248,7 @@ Next we need to implement the business logic in the `ballerina_petstore_impl.bal
 
 ### Implement the business logic for petstore 
 
-Now we have the Ballerina web service for the give `petstore.json` Swagger file. Next task is to implement the business logic for functionality of each resource. The Ballerina Swagger generator has generated `ballerina_petstore_impl.bal` file inside the `open-api-based-service/guide/petstore`. We need to fill the `ballerina_petstore_impl.bal` as per the requirement. For simplicity, we will use an in-memory map to store the pet data. The following code is the completed pet store web service implementation. 
+Now we have the Ballerina web service for the given `petstore.json` OpenAPI file. Next task is to implement the business logic for functionality of each resource. The Ballerina OpenAPI generator has generated `ballerina_petstore_impl.bal` file inside the `open-api-based-service/guide/petstore` directory. We need to fill the `ballerina_petstore_impl.bal` as per the requirement. For simplicity, we will use an in-memory map to store the pet data. The following code is the completed pet store web service implementation.
```
Comment (praneesha) [inline README.md]:
```suggestion
After creating the Ballerina web service for the given `petstore.json` OpenAPI file, next, the business logic should be implemented for the functionality of each resource. 

To do this, fill the `open-api-based-service/guide/petstore/`ballerina_petstore_impl.bal` file generated by the Ballerina OpenAPI generator with the required details as preferred. 

For simplicity, this guide uses an in-memory map to store the pet data. The following code is the complete implementation of the pet store web service. 
```

**Pull request closed without merging**